runtests.py

```diff
@@ -11,9 +11,5 @@
 
 django.setup()
 
-from django.test.utils import setup_test_environment
-setup_test_environment()
```

Comment:
`setup_test_environment()` gets called when running the `test` management command. In Django 1.11, an error gets raised if `setup_test_environment()` is called more than once, so this call was removed.